src/orange/orangeinp/detail/PolygonUtils.hh

```diff
@@ -119,6 +87,84 @@ is_same_orientation(Orientation a, Orientation b, bool degen_ok = false)
     return (a == b);
 }
 
+//---------------------------------------------------------------------------//
+/*!
+ * Functor for calculating orientation with a tolerance for collinearity.
+ *
+ * Collinearity is based on a supplied absolute tolerance. For three ordered
+ * points a, b, c, point b is collinear if the displacement, d, is less than
+ * the absolute tolerance.
+ *
+ *              b
+ *             . .
+ *           .  .  .
+ *         .    .    .
+ *       .      . d    .
+ *     .  t     .        .
+ *   a . . . . . . . . . . c
+ *
+ * The displacement is calculated as follows.
+ *
+ * Let:
+ * u = b - a
+ * v = c - a
```

Comment:
```suggestion
 * Let: \f[
 * \vec u = \vec b - \vec a
 * \vec v = \vec c - \vec a
```

Reply:
Needs to be either two separate equations or use the split environment. Also I'm not too worried about equation rendering in detail classes; it's most important for classes rendered in the user manual.

Reply:
@sethrj why does this need to be two separate equations?